onnxruntime/core/session/inference_session.cc

```diff
@@ -1499,7 +1503,7 @@ common::Status InferenceSession::ValidateInputs(const std::vector<std::string>&
         return ORT_MAKE_STATUS(ONNXRUNTIME, INVALID_ARGUMENT, "Input with name: ", feed_name,
```

Comment:
TODO: Need to legalize receiving inputs that are sequence tensors for expected types that are sequence(tensors) and optional(sequence(tensors))